### PR TITLE
data: RoHit4100

### PR DESCRIPTION
**Pull request by RoHit4100**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2800"></a>

### CODE REVIEW

File: data/RoHit4100.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Rohit Udamale",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "**Student**, Currently learning web3 and react, Want to contribute to real life projects",
+  "avatar": "https://avatars.githubusercontent.com/u/70364267?s=400&u=6ae48f02ca4bf044f149a8d33630d3f1fa117e65&v=4",
```
Comment (senali-d) [inline data/RoHit4100.json:6]: Update image url to short form.
```suggestion
  "avatar": "https://github.com/RoHit4100.png",
```

File: data/RoHit4100.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/rohit__udamale",
+      "icon": "FaTwitter"
+    },
+  {
+      "name": "Follow me on Linkden",
+      "url": "https://www.linkedin.com/in/rohit-udamale-5b3023236",
+      "icon": "FaLinkedin"
+    },
```
Comment (senali-d) [inline data/RoHit4100.json:27]:
```suggestion
    }
```

**Review by senali-d — COMMENTED** [review]
Hi @RoHit4100,
There is some inline comments. Please have a look.

**Comment by RoHit4100** [comment]
> 

Thank You @senali-d, for the suggestions they were helpfull to me!

**Review by senali-d — COMMENTED** [review]
Look Good. 👍

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by RoHit4100** [comment]
@Pradumnasaraf Thanku🚀

**Pull request merged**